.gitignore

```diff
@@ -46,4 +46,9 @@ target/
 .mule/**
 .mule/**/*
 .DS_Store
-velocity.log
\ No newline at end of file
+velocity.log
+
+# ---------------#
+# Mule Specific  #
+# ---------------#
+src/main/resources/weave/*
```

Comment:
Why are we ignoring things from src/main/resources?

Reply:
I wanted to ignore those autogenerated files that the IDE creates into `src/main/resources/weave/autogenerated` folder. Updating it due `/autogenerated` is missing in the path.